app/views/meetings/show.html.erb

```diff
@@ -1,19 +1,61 @@
-<p id="notice"><%= notice %></p>
 
-<p>
-  <strong>Start time:</strong>
-  <%= @meeting.start_time %>
-</p>
+<div class="row">
+  <div class="col-lg-10 offset-lg-1">
+    <% if notice%>
+      <p id="notice"><%= notice %></p>
+    <% end%>
+    <%= link_to 'Back to class overview',
+                resource_path(@meeting.resource),
+                class: "text-decoration-none" %>
+    <% if current_enrollment.teacher?%>
+      <div class="card">
+        <div class="card-header">
+          <h1 class="h3 d-flex justify-content-between">
+            <span>
+              <%= @meeting.start_time.strftime("%B %e")%>
+            </span>
 
-<p>
-  <strong>End time:</strong>
-  <%= @meeting.end_time %>
-</p>
+            <span>
+              <%= link_to edit_resource_meeting_path(@meeting.resource, @meeting),
+                          class: "text-warning text-decoration-none" do %>
+                <i class="fas fa-edit"></i>
+              <% end %>
+              <%= link_to resource_meeting_path(@meeting.resource, @meeting),
+                          class: "text-danger text-decoration-none",
+                          method: :delete,
+                          data: { confirm: 'Are you sure you want to delete this meeting? Attendance records and content will be permanently deleted.' } do %>
+                <i class="fas fa-trash-alt"></i>
+              <% end%>
+            </span>
 
-<p>
-  <strong>Resource:</strong>
-  <%= @meeting.resource_id %>
-</p>
-
-<%= link_to 'Edit', edit_resource_meeting_path(@meeting) %> |
-<%= link_to 'Back', resource_meetings_path(@meeting.resource) %>
+            <span>
+              <%= @meeting.start_time.strftime("%l:%M %P") %>
+            </span>
+          </h1>
+        </div>
+      </div>
+    <% else %>
+      <%= render @meeting %>
+    <% end %>
+    
+    <div class="card">
+      <div class="card-header">
+        <h1 class="h3 d-flex justify-content-between">
+          <span>
+            Class Content
+          </span>
+        </h1>
+      </div>
+      
+      <div class="card-body">
+        <% if @meeting.in?(current_submission.approved_meetings) || current_enrollment.teacher? %>
+          <%= @content.to_html.html_safe %>
```

Comment:
@LoganDSPrice since you're rendering `@content` as HTML, won't embeds just work if you copy paste it? Although, it probably won't work if Checkins is being rendering in an `iframe` already  🤔

Reply:
https://checkins-staging-pr-93.herokuapp.com/resources/1/meetings/2
<img width="955" alt="checkins-video-embed-question" src="https://user-images.githubusercontent.com/17581658/73504572-77f5b180-4395-11ea-9b9b-65f17e128fd0.png">

Reply:
The idea is to be able to edit the `content` field of a meeting. That would would contain markdown. So the idea would be to embed it in the markdown...

I think I remember Raghu saying there's a way he's doing this in Chapters.

Alternatively, we could make a Videos table and a join table with meetings.

Reply:
Oh wait... I see it's working [here](https://checkins-staging-pr-93.herokuapp.com/resources/1/meetings/2).

Reply:
I kinda forgot you can just put html straight into markdown...

Reply:
I might be misunderstanding you, but you can embed a video in the `content` field of a meeting. I attempted to demonstrate that with the screenshot.

Here's the raw input in the form for a Meeting

<img width="747" alt="checkins-markdown-embed" src="https://user-images.githubusercontent.com/17581658/73505132-3b2aba00-4397-11ea-9f20-26cc5c8b92b8.png">

Reply:
Yeah that's what I was thinking. I don't know if there was a different way you wanted to handle it though.